src/runtime/hexagon_remote/halide_hexagon_remote.idl

```diff
@@ -17,10 +17,13 @@ interface halide_hexagon_remote
     long power_hvx_off();
 
     // Routine to run a pipeline on the remote side.
-    long run(in handle_t module_ptr, in handle_t symbol,
-             in sequence<buffer> input_buffers,
-             rout sequence<buffer> output_buffers,
-             in sequence<buffer> input_scalars);
+    // v2: Pass all scalars and small input buffers in a single
```

Comment:
I just took a quick glance at this and will do a full review tomorrow. I think it would help to expand this comment to define what how "small input buffers" are defined. The juxtaposition of "small input buffers" in this sentence and "null input buffers" in the next is unexpected, but perhaps that will make sense once I understand the code.